packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3978,6 +3981,20 @@ class ExcludeSemantics extends SingleChildRenderObjectWidget {
   }
 }
 
+class _ExcludeSemanticsElement extends SingleChildRenderObjectElement {
+  _ExcludeSemanticsElement(ExcludeSemantics widget) : super(widget);
+
+  @override
+  ExcludeSemantics get widget => super.widget;
+
+  @override
+  void visitChildrenForSemantics(ElementVisitor visitor) {
+    if (widget.excluding)
+      return;
```

Comment:
i'm pretty sure this will exclude stuff that you want to be considered onstage. For example it'll exclude the avatar in a chip, which I would expect people to want to be able to find in tests.